src/organization/screens/organization-profile.screen.js

```diff
@@ -47,16 +48,12 @@ const actionCreators = {
 
 const actions = dispatch => bindActionCreators(actionCreators, dispatch);
 
-const styles = StyleSheet.create({
-  listTitle: {
-    color: colors.black,
-    ...fonts.fontPrimary,
-  },
-  listSubTitle: {
+const DescriptionListItem = styled(ListItem).attrs({
+  subtitleStyle: {
     color: colors.greyDark,
     ...fonts.fontPrimary,
   },
-});
+})``;
```

Comment:
Can we remove this? Was probably just added by `prettier` for some reason, but I think you can safely get rid of it.

Reply:
It's needed by styled components when using `attrs()`

Reply:
Ahh alright, yes you're right.